dbms/include/DB/DataStreams/JSONCompactRowOutputStream.h

```diff
@@ -14,7 +14,7 @@ namespace DB
 class JSONCompactRowOutputStream : public JSONRowOutputStream
 {
 public:
-	JSONCompactRowOutputStream(WriteBuffer & ostr_, const Block & sample_, bool write_statistics_);
+	JSONCompactRowOutputStream(WriteBuffer & ostr_, const Block & sample_, bool write_statistics_, bool force_quoting_);
```

Comment:
Let rename `force_quoting` to `force_quoting_64bit_integers` everywhere.
Because otherwise developers may think that it will quote all data types (or all numbers).
Long name is no problem.